operation/operation.c

```diff
@@ -39,15 +37,15 @@ void set_opto_fork_irq() {
                                        &irq_event);
 }
 
-//set piezo interrupt detection with callback event
+// set piezo interrupt detection with callback event
 void set_piezo_irq() {
```

Comment:
I'm not suggesting this, just offering a though:
these two functions could be combined as one...?

Reply:
I would prefer to keep it separate